### PR TITLE
com.giuspen.cherrytree -> net.giuspen.cherrytree

### PR DESCRIPTION
**Pull request by giuspen**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: net.giuspen.cherrytree.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/fmtlib/fmt/releases/download/7.1.3/fmt-7.1.3.zip",
+                    "sha256" : "5d98c504d0205f912e22449ecdea776b78ce0bb096927334f80781e720084c9f"
+                }
+            ]
+        },
+        {
+            "name": "spdlog",
+            "buildsystem": "cmake",
```
Comment (hfiguiere) [inline net.giuspen.cherrytree.json:208]:
```suggestion
            "buildsystem": "cmake-ninja",
```

File: net.giuspen.cherrytree.json [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+                {
+                    "type" : "archive",
+                    "url" : "https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.7.tar.xz",
+                    "sha256" : "3fc79408ae1d84b406922fa9319ce005631c95ca0f34b205fad867e8b30e45b1"
+                }
+            ]
+        },
+        {
+            "name": "fmt",
+            "buildsystem": "cmake",
```
Comment (hfiguiere) [inline net.giuspen.cherrytree.json:197]:
```suggestion
            "buildsystem": "cmake-ninja",
```

File: net.giuspen.cherrytree.json [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+                {
+                    "type" : "archive",
+                    "url" : "https://ftp.acc.umu.se/pub/GNOME/sources/gspell/1.9/gspell-1.9.1.tar.xz",
+                    "sha256" : "dcbb769dfdde8e3c0a8ed3102ce7e661abbf7ddf85df08b29915e92cd723abdd"
+                }
+            ]
+        },
+        {
+            "name": "uchardet",
+            "buildsystem": "cmake",
```
Comment (hfiguiere) [inline net.giuspen.cherrytree.json:186]:
```suggestion
            "buildsystem": "cmake-ninja",
```

**Comment by hfiguiere** [comment]
bot, build net.giuspen.cherrytree

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: net.giuspen.cherrytree.json [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+    "sdk": "org.gnome.Sdk",
+    "command": "cherrytree",
+    "rename-desktop-file": "cherrytree.desktop",
+    "rename-icon": "cherrytree",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```
Comment (hfiguiere) [inline net.giuspen.cherrytree.json:14]: that is a very broad filesystem permission

File: net.giuspen.cherrytree.json [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "cherrytree",
+    "rename-desktop-file": "cherrytree.desktop",
+    "rename-icon": "cherrytree",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
```
Comment (hfiguiere) [inline net.giuspen.cherrytree.json:12]: does it support wayland?

**Comment by hfiguiere** [comment]
what's the relation with https://flathub.org/apps/details/com.giuspen.cherrytree

**Comment by barthalion** [comment]
bot, build net.giuspen.cherrytree

**Comment by hfiguiere** [comment]
> what's the relation with https://flathub.org/apps/details/com.giuspen.cherrytree

Still that question is unanswered.

**Comment by barthalion** [comment]
It's obviously a rename due to domain change.

**Comment by hfiguiere** [comment]
it's not specified in the submission where there is the question acknowledged by the OP. Hence my question.

**Comment by barthalion** [comment]
@giuspen Please remember to mark the old ID as EOL: https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

/merge

**Comment by giuspen** [comment]
Thanks @barthalion I will mark the old ID to EOL. @hfiguiere yes it is a rename due to domain change.

**Pull request closed without merging**